Multitrack-Template.lua

```diff
@@ -0,0 +1,205 @@
+--[[
+Multi-track re-record script template.
+
+This template was created from DarkKobold's Multitrack LUA script created
+for TAS of TMNT Arcade and The Simpsons Arcade.
+
+The intent of this template script is to make multiplayer TAS'ing more
+approachable to newbies - empowering them to quickly create a
+multi-track re-recording rig for the game they plan to TAS
+--]]
+
+-- Max number of lines to display in LUA overlay
```

Comment:
I created constants from all the hardcoded values throughout the script - makes it much easier for new people to come along and adjust the script to quickly accommodate their game - without having do much further down